### PR TITLE
added-snyk

### PR DESCRIPTION
**Pull request by mayank0202**
tflint check is not passing because of api limit got exceeded..

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -77,6 +77,7 @@ resource "kubernetes_deployment" "default" {
               read_only         = lookup(csi_secret_volume.value, "read_only", null)
               driver            = lookup(csi_secret_volume.value, "driver", null)
               volume_attributes = lookup(csi_secret_volume.value, "volume_attributes", null)
+              volume_handle     = lookup(csi_secret_volume.value, "volume_handle", null)
```
Comment (jamescrowley321) [inline main.tf:80]: Why is this added? Don't touch this unless you have validated that secrets mounting still works correctly.

**Review by jamescrowley321 — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (mayank0202) [reply main.tf:80]: I ran terraform validate and it showed me an error that volume_handle is missing

Comment (jamescrowley321) [reply main.tf:80]: There is an issue with the provider version then. That param has a history of problems and shouldn't be added.

Comment (mayank0202) [reply main.tf:80]: In providers we are using an old version of kubernetes provider

Comment (jamescrowley321) [reply main.tf:80]: Upgrade to the latest version and let's see what happens. Coordinate with @tsaucier-sf on testing.

Comment (mayank0202) [reply main.tf:80]: sure

**Review by jamescrowley321 — APPROVED** [review]
